.gitignore

```diff
@@ -1,6 +1,6 @@
 .now
 node_modules/
-.env
+.env*
```

Comment:
What other examples of `env` files are affected by this filter?

Reply:
I was trying to use `.env.production` and other files to deploy with now, but ultimately realized that `next.config.js` is the best way to handle this.